java/google/registry/model/reporting/HistoryEntry.java

```diff
@@ -63,6 +63,10 @@
     HOST_PENDING_DELETE,
     HOST_UPDATE,
     /**
+     * Resource was created by an escrow file import.
+     */
+    RDE_IMPORT,
```

Comment:
for reference, here are the notes I took of the conversation:

https://groups.google.com/forum/#!msg/nomulus-discuss/o2yI9nR4TF0/eCB66R4wFQAJ;context-place=searchin/nomulus-discuss/history$20entry$20type

And the relevant section from the notes:

A new “escrow import” history entry type should be added, with the contents being the xml snippet from the escrow file (typically the contents are the payload of the epp message)